tests/integration/tests/menu.js

```diff
@@ -40,7 +40,7 @@ test('menu open favorite', async () => {
   page.click('.menu__panel__action:nth-child(3)');
 
   expect(await exists(page, '.favorite_panel')).toBeTruthy();
-});
+}, 10000);
```

Comment:
This test often fail on my computer due to a too short duration

Reply:
Maybe we can augment the timeout for every test in the global config. That would be better than having this 10s here in a test case that doesn't seem so costly. 
I'm curious, how long does it take to run?

Reply:
I only know it's between 5000 and 10000ms :turtle:

Reply:
Tests on your computer seem to behave really strangely… Are you mining bitcoins aside? :smile:
Anyway, there is no reason *this* case in particular should take longer. I'm pretty sure if we change the order or add some other test case it will be another one which fails, so there is no point to keep this `10000` here.